build.assets/makefiles/master/k8s-master/kube-kubelet.service

```diff
@@ -9,7 +9,7 @@ EnvironmentFile=/run/dns.env
 EnvironmentFile=-/run/kubernetes-reserved.env
 ExecStartPre=/bin/systemctl is-active docker.service
 ExecStart=/usr/bin/kubelet \
-        --root-dir=/var/lib/gravity/planet/kubelet \
+        --root-dir=/var/lib/kubelet \
```

Comment:
Didn't we move this directory under /var/lib/gravity?

Reply:
Yes, we did but I wanted to keep this explicitly under `/var/lib/kubelet` (as this is the default location in kubelet source code) due to its unfortunate hard-coding for some of the paths which then interfere when SELinux is enabled - hence `/var/lib/gravity/planet/kubelet` won't work.